### PR TITLE
identify: add page

### PR DESCRIPTION
**Pull request by UltimatePea**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
I corrected your page to use the correct formatting.

File: pages/common/identify.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# identify
+
+> Image magick command.
+
+- Collect dimensions of all jpeg files under current directory:
+
+`identify -format "%f,%w,%h\n" *.jpg > filelist.csv`
```
Comment (zdroid) [inline pages/common/identify.md:7]:
```suggestion
`identify -format "%f,%w,%h\n" *.{{jpg}} > {{filelist.csv}}`
```

File: pages/common/identify.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# identify
+
+> Image magick command.
```
Comment (andrik) [inline pages/common/identify.md:3]:
```suggestion
>  Command line utility of Image Magick project to describe the format and characteristics of one or more image files.
> More information: <https://imagemagick.org/script/identify.php>.
```

File: pages/common/identify.md (hunk 1 above)
Comment (sbrl) [reply pages/common/identify.md:7]: `filelist.csv` → `path/to/filelist.csv`, @zdroid?

**Comment by sbrl** [comment]
Nice one! Very useful command there.

**Comment by ivanhercaz** [comment]
> The [build](https://travis-ci.org/tldr-pages/tldr/builds/687290045) for this PR failed with the following error(s):
> 
> ```
> pages/common/identify.md:3: TLDR002 A single space should precede a sentence
> ```
> 
> Please fix the error(s) and push again.

As it is was just a minor detail, I just fix it, @UltimatePea. Thank you very much for you contribution and welcome to tldr-pages!

it is just necessary @zdroid review it and accept it :tada:

**Pull request merged**